libraries/botbuilder-dialogs/src/dialogHelper.ts

```diff
@@ -18,6 +18,7 @@ import { Dialog, DialogTurnStatus } from './dialog';
 import { DialogEvents } from './dialogEvents';
 import { DialogSet } from './dialogSet';
 import { AuthConstants, GovConstants, isSkillClaim } from './prompts/skillsHelpers';
+import { EndOfConversationCodes } from '../../botframework-schema'
```

Comment:
This is a bad import; it should instead be from `'botbuilder-core'`

Reply:
changed to 'botbuilder-core'